### PR TITLE
Add description overlays to the demos

### PR DESCRIPTION
**Pull request by joshmarinacci**

### CODE REVIEW

**Comment by TrevorFSmith** [comment]
Hey, we're now working on the 'develop' branch instead of directly on master so that we can sync the iOS and and polyfill code as it rolls out through the Apple App Store.

So, please change these PRs so that they're against 'develop'.

File: examples/ar_anchors/index.html [hunk 1]
```diff
@@ -32,6 +32,17 @@
 				transform: translate(-50%, -50%);
 				font-size: 20px;
 			}
+			#description {
+			    pointer-events: auto;
+			    font-family: sans-serif;
+			    padding: 1em;
+			    background-color:rgba(255,255,255,0.7);
+			    -webkit-backdrop-filter: blur(5px);
```
Comment (TrevorFSmith) [inline examples/ar_anchors/index.html]: These samples need to work on more than just the WebXR Viewer, so please include the non-prefixed CSS.

File: examples/boombox/index.html [hunk 2]
```diff
@@ -24,6 +24,17 @@
...
 			}
+			#description {
+			    pointer-events: auto;
+			    font-family: sans-serif;
+			    padding: 1em;
+			    background-color:rgba(255,255,255,0.7);
+			    -webkit-backdrop-filter: blur(5px);
+			    position:absolute;
+			    bottom: 0px;
+			    left:0px;
```
Comment (TrevorFSmith) [inline examples/boombox/index.html]: Should both 'left' and 'right' be defined?

File: examples/peoples/index.html [hunk 3]
```diff
@@ -49,6 +60,11 @@
 	</head>
 	<body>
 		<div id="target" />
+        <div onclick="hideMe(this)" id="description">
```
Comment (TrevorFSmith) [inline examples/peoples/index.html]: Please follow the rest of the code and use tabs, not spaces for indentation.

Comment (blairmacintyre) [reply examples/peoples/index.html]: "It's not me, it's my editor"

Comment (blairmacintyre) [reply examples/peoples/index.html]: (how many spaces do you map tabs to? 2? 3? 4?)

File: examples/boombox/index.html (hunk 2 above)
Comment (blairmacintyre) [reply examples/boombox/index.html]: no, I don't think so.

Comment (joshmarinacci) [reply examples/boombox/index.html]: yes. I thought I did that everywhere. It ensures the div spans the whole screen.

File: examples/peoples/index.html (hunk 3 above)
Comment (joshmarinacci) [reply examples/peoples/index.html]: hmm. weird. i'm doing this live on my server so it wasn't through my normal editor.

Comment (TrevorFSmith) [reply examples/peoples/index.html]: Why does it matter how many spaces I map tabs to? Use tab characters, not spaces, for indentation.

File: examples/boombox/index.html (hunk 2 above)
Comment (TrevorFSmith) [reply examples/boombox/index.html]: Hmmm. The MDN docs says that having both is an over-specification and only one is honored. Does it actually work?

Here's the MDN link: https://developer.mozilla.org/en-US/docs/Web/CSS/left

Comment (joshmarinacci) [reply examples/boombox/index.html]: Wow. That's so weird. I've never looked at the spec for left before.  Well, regardless of what the spec says, yet it works. When you absolutely position a div you can set the left and right but leave the width undefined, and the div will size itself based on the parent context.

**Pull request closed without merging**